plop-templates/example/README.md

```diff
@@ -0,0 +1,41 @@
+# -- PLOP TITLE HERE -- example
+
+This example shows how -------------------------
+
+## Demo
+
+https://-----------------------.vercel.app
```

Comment:
Probably we can do this line the same width of the above one so we can find and replace it

Reply:
yeah, I was thinking about this. But sometimes we have a different URL and when it's auto-populated we will take it for granted and with those huge dashes, it's obvious to update this. But yeah I can automate it. The question is if code reviewer will always check this (before and after merge).

Reply:
Probably the project won't be created until merge, so this can be modified later in case than needed, if we don't fill it we still have to come back to this, I preffer to come and change a few chars sometimes but don't change it most of the times

Reply:
done